sdk/synapse/Azure.Analytics.Synapse.AccessControl/README.md

```diff
@@ -85,67 +85,29 @@ The Azure.Analytics.Synapse.AccessControl package supports synchronous and async
 
 To interact with Azure Synapse, you need to instantiate a `RoleAssignmentsClient` and a `RoleDefinitionsClient`. It requires an endpoint URL and a `TokenCredential`.
 
-```C# Snippet:CreateAccessControlClient
```

Comment:
Snippets will be restored in https://github.com/azure/azure-sdk-for-net/issues/24395